content/en/docs/tasks/observability/distributed-tracing/configurability/telemetry-api/index.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Configuring tracing using the Telemetry API
+description: How to configure tracing options using the Telemetry API.
+weight: 10
+keywords: [telemetry,tracing]
+owner: istio/wg-policies-and-telemetry-maintainers
+test: no
+status: Experimental
+---
+
+Istio provides a Telemetry API that enables flexible configuration of tracing behavior. The
+Telemetry API offers controls over tracing options such as sampling rates and custom tags for individual
+spans, as well as backend provider selection.
+
+## Before you begin
+
+1.  Ensure that your applications propagate tracing headers as described [here](/docs/tasks/observability/distributed-tracing/overview/).
+1.  Follow the tracing installation guide located under [Integrations](/docs/ops/integrations/) based on your preferred tracing backend
+    to install the appropriate tracing provider.
```

Comment:
```suggestion
1.  Follow the tracing installation guide located under [Integrations](/docs/ops/integrations/)
    to install your preferred tracing provider.
```

Reply:
just "Follow the guide", you say tracing later in the sentence?